tests/rest/client/test_auth.py

```diff
@@ -619,7 +632,12 @@ def test_token_refresh(self):
             refresh_response.json_body["refresh_token"],
         )
 
-    @override_config({"refreshable_access_token_lifetime": "1m"})
+    @override_config(
+        {
+            "refreshable_access_token_lifetime": "1m",
+            "nonrefreshable_access_token_lifetime": "10m",
```

Comment:
Not sure I understand why this is needed. Isn't this test only concerned with refreshable access tokens?

Reply:
Ah oups, I went back and forth a bit on whether to roll it in or make it a separate test, then left that there. Reverted.